style/themes/lcars-picard.css

```
@@ -0,0 +1,5906 @@
+  background-color: var(--color-primary);
+  color: #fff !important;
+}
+
+.btn-whitelist > svg.svg-inline--fa,
+.btn-blacklist > svg.svg-inline--fa,
+.btn-audit > svg.svg-inline--fa,
+.btn-xs.btn-whitelist > svg.svg-inline--fa,
+.btn-xs.btn-blacklist > svg.svg-inline--fa,
+.btn-xs.btn-audit > svg.svg-inline--fa {
+  position: absolute;
+  inset: 0 auto 0 0;
+  width: calc(var(--gap) * 2);
+  height: calc(var(--gap) * 2 + var(--border-width));
+  padding: 0 var(--gap50);
+  border-right: var(--border-width) solid var(--color-background);
+}
+
+.btn-xs.btn-whitelist > svg.svg-inline--fa,
+.btn-xs.btn-blacklist > svg.svg-inline--fa,
+.btn-xs.btn-audit > svg.svg-inline--fa {
+  height: calc(var(--gap) * 1.5 + var(--border-width));
+}
+
+.btn-xs.btn-whitelist,
+.btn-xs.btn-blacklist {
+  margin-right: var(--gap25);
+}
+
+.btn-success,
+.bs-actionsbox > button.btn-success,
+#add-group .btn-toolbar #add2white {
+  background-color: var(--color-success-dark);
+  color: var(--color-success);
+}
+
+.btn-success:hover,
+.bs-actionsbox > button.btn-success:hover,
+#add-group .btn-toolbar #add2white:hover {
+  background-color: var(--color-link) !important;
+  color: #fff;
+}
+
+#resetButton,
+.btn-danger {
+  background-color: var(--color-red-alert);
+  color: #fff;
+}
+
+#resetButton:hover,
+.btn-danger:hover {
+  --pulse-color: #fff;
+  --pulse-color-background: var(--color-red-alert);
+  color: var(--color-background);
+  animation: pulseBg 2s ease-in-out infinite;
+}
+
+#resetButton {
+  color: #fff !important;
+  margin-top: var(--gap);
+  min-width: calc(var(--gap) * 8);
+}
+
+.btn-xs {
+  padding: 1px 5px;
+  border-radius: 5px;
+  min-width: auto;
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link,
+.login-footer .btn-default,
+.login-box button.btn-primary {
+  background-color: var(--color-disabled) !important;
+  color: var(--color-link);
+  padding: var(--gap50) var(--gap) var(--gap50) calc(var(--gap) * 3.5);
+  position: relative;
+  text-align: right;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link:hover,
+.login-footer .btn-default:hover,
+.login-box button.btn-primary:hover {
+  background-color: var(--color-primary) !important;
+  color: #fff;
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link svg,
+.login-footer .btn-default svg,
+.login-box button.btn-primary svg {
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.35);
+  padding: calc(var(--gap75));
+  width: calc(var(--gap) * 3);
+  height: 36px !important;
+  inset: 0 auto 0 0;
+  border-right: calc(var(--border-width) * 2) solid var(--color-background);
+  border-radius: 18px 0 0 18px;
+}
+
+/* CHECKBOXES, RADIOS
+  ========================================================== */
+/* body[class*="lcars"][class*="lcars"] is a small dirty trick to avoid even more deep "!important hell" */
+
+body[class*="lcars"][class*="lcars"] [class*="icheck-"] > label {
```

Why did you add the attribute selector (`[class*="lcars"]`) twice?

```suggestion
body[class*="lcars"] [class*="icheck-"] > label {
```

---

I made a comment there to clarify -  `/* body[class*="lcars"][class*="lcars"] is a small dirty trick to avoid even more deep "!important hell" */`

The `pihole.css` contains several lines of code with `body:not(.lcars) ...`. `:not` adds specificity, so I needed to do the same, but without depending on another class or putting !important everywhere.

---

I understand the hack, but are you sure it's needed? 
You already increased the specificity with `body[class*="lcars"]`.

I will test again another time, but apparently it works without the hack.

---

Nightly, with `lcars` or `lcars-picard` class. If I remove one of the `[class*="lcars"]` on line 398 - it breaks the checkboxes.

---

@MichalSvatos

I finally found time to test your PR again.

When I use the simplified selector, I saw the issue only in "Domains" page (do you see it in other places?).

I noticed there are some things contributing for the issue, but we can fix them instead of adding a new hack or a lot of `!important` rules.

I pushed a new branch with the changes. I will wait the team approval and them we can simplify the selectors here.

---

Yes, it looks like that at the end only the "Domains" page cause an issue.

This PR is based on `devel` not `development-v6`, are you sure that I should rebase?

(I'm sorry for such a delay.)

---

Sorry... it was a distraction when I typed.

New code is in **`devel`**.